ui/src/specs/test.code.js

```diff
@@ -0,0 +1,38 @@
+/* global describe, it, chai */
```

Comment:
Maybe at some point, we should overhaul the gulp build to compile/bundle the JavaScript code.
That would allow us to write modern ES6 code (with export/import syntax). That would probably make things easier to unit tests with Node.js.

Alternatively, we could use Puppeteer to automate testing using the browser.

For now, you can open this page manually using a local HTTP server:

![mocha-browser](https://user-images.githubusercontent.com/333276/98934285-206aa180-24e2-11eb-89a8-d8095565dc4e.png)